code/datums/traits/neutral.dm

```diff
@@ -106,3 +106,46 @@
 /datum/quirk/monochromatic/remove()
 	if(quirk_holder)
 		quirk_holder.remove_client_colour(/datum/client_colour/monochrome)
+
+/datum/quirk/felinid
+	name = "Body Modder"
+	desc = "You have undergone unethical body modification to look like a catperson."
+	value = 0
+	mob_trait = TRAIT_FELINID
+	gain_text = "<span class='notice'>You have a cute pair of ears and a tail!</span>"
+	lose_text = "<span class='notice'>You've outgrown your catperson phase</span>"
+
+/datum/quirk/felinid/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
+		var/datum/species/species = H.dna.species
+		species.liked_food |= DAIRY
+		species.liked_food |= MEAT
+		species.disliked_food |= SUGAR
+		species.disliked_food |= VEGETABLES
```

Comment:
This alone is more snowflake code than the entire species itself and not how species should be used.

Reply:
Just works like other taste change prs

Reply:
The other taste change quirks have no other way to do it. This one has a much much better way to do it; by having them as a species.